spec/controllers/publishers/vacancies/application_controller_spec.rb

```diff
@@ -4,23 +4,17 @@
   describe "#update_google_index" do
     let!(:vacancy) { create(:vacancy) }
 
-    context "when in production" do
-      before(:each) do
-        allow(Rails).to receive(:env)
-          .and_return(ActiveSupport::StringInquirer.new("production"))
-      end
+    context "when DisableExpensiveJobs is not enabled" do
+      before(:each) { allow(DisableExpensiveJobs).to receive(:enabled?).and_return(false) }
```

Comment:
I'm pretty sure `before` is the same as `before(:each)`

Reply:
copy and 🍝